docs/en/guide/gridview.md

```diff
@@ -0,0 +1,240 @@
+# Grid View
+
+GridView is a powerful data presentation widget that displays data in a highly customizable grid format.
+It supports features such as:
+
+- Pagination
+- Sorting
+- Filtering
+- Custom column rendering
+- Action buttons
+- Row customization
+- Header and footer sections
+
+## Basic Usage
+
+Here's a basic example of using GridView:
+
+```php
+<?php
+use Yiisoft\Yii\DataView\GridView;
+use Yiisoft\Yii\DataView\Column\DataColumn;
+use Yiisoft\Yii\DataView\Column\ActionColumn;
+use Yiisoft\Yii\DataView\DataReader\DataReaderInterface;
+
+$dataReader = new DataReader($query);
+?>
+
+
+<?= GridView::widget()
+    ->data($dataReader)
+    ->columns(
+        new DataColumn(property: 'id'),
+        new DataColumn(property: 'title', header: 'Post Title'),
+        new DataColumn(property: 'created_at')
+    )
+?>
+```
+
+It is getting data from data reader and rendering it according to configuration passed in `columns()`.
+
+## Column Types
+
+GridView supports several types of columns out of the box.
+
+### Data Column
+
+DataColumn is the most commonly used column type. It displays model attribute values:
+
+```php
+use Yiisoft\Yii\DataView\Column\DataColumn;
+
+$column = new DataColumn(
+    property: 'title',
+    header: 'Post Title',
+    withSorting: true
+)
+```
+
+### Action Column
+
+ActionColumn displays action buttons (e.g., view, edit, delete):
+
+```php
+use Yiisoft\Yii\DataView\Column\ActionColumn;
+use Yiisoft\Html\Html;
+
+$column = new ActionColumn(
+    buttons: [
+        'view' => new \Yiisoft\Yii\DataView\Column\ActionButton(
+            'View',
+            static function (array|object $data, DataContext $context) { 
+                return '/posts/view' . $data->id; 
+            }
+        ),
+        'edit' => function (string $url): string {
+            return (string) Html::a('Edit', $url);
+        },
+        'delete' => function (string $url): string {
+            return (string) Html::a('Delete', $url);
+        },
+    ],
+    urlCreator: function ($action, $model) {
+        return "/posts/$action/" . $model->getId();
+    },
+);
+```
+
+### Checkbox Column
+
+CheckboxColumn adds checkboxes for row selection:
+
+```php
+use Yiisoft\Yii\DataView\Column\CheckboxColumn;
+
+$column = new CheckboxColumn(
+    name: 'selection',
+    multiple: true,
+);
+```
+
+## Filtering
+
+// TODO: fix it!
```

Comment:
Fix or move to issue

Reply:
https://github.com/yiisoft/yii-dataview/issues/241